src/app/shared/dovetail/dovetail-audio.ts

```diff
@@ -43,7 +38,7 @@ const PROXIED_EVENTS = [
 ];
 
 export class DovetailAudio extends ExtendableAudio {
-  private arrangement: DovetailArrangement = {entries: []};
+  private arrangement: DovetailSegment[] = [];
```

Comment:
As far as I could tell, there wasn't much point keeping the arrangement as an object.  So I moved entries -> DovetailSegment[], and called it good.

Reply:
reasonable